packages/oscal-react-library/src/components/HoverablePopover.tsx

```diff
@@ -0,0 +1,88 @@
+import { Box, Popover } from "@mui/material";
+import React from "react";
+import { OSCALMarkupLine, OSCALMarkupMultiLine } from "./OSCALMarkupProse";
+
+export interface HoverablePopoverProps {
+  /**
+   * The content of the popover.
+   */
+  popoverContent: React.ReactNode;
+  /**
+   * Content to to hover over. This must fit within a span.
+   */
```

Comment:
Fair enough. I had thought it worth mentioning that it is placed within a `span` since I couldn't think of a better way of restricting it. I will remove that part of the comment.